src/openvic-simulation/dataloader/Dataloader.cpp

```diff
@@ -587,6 +587,7 @@ bool Dataloader::load_defines(GameManager& game_manager) const {
 	static const fs::path units_directory = "units";
 
 	static const fs::path buildings_file = "common/buildings.txt";
+	static const fs::path bookmark_file = "common/bookmarks.txt";
```

Comment:
I feel like rearranging this is kinda pointless in this PR, like if we need to rearrange this stuff, we should prefer a PR focused on that, but whatever I guess